lib/kwtool/crawlers.ex

```diff
@@ -26,7 +26,13 @@ defmodule Kwtool.Crawlers do
     end
   end
 
-  defp create_keyword(attrs) do
+  def list_keywords do
+    Repo.all(Keyword)
+  end
+
+  def get_keyword!(id), do: Repo.get!(Keyword, id)
```

Comment:
Do we need to fetch the keyword that belongs to the current user?

Reply:
yup, sure, but i will do it in the backend ticket.

Reply:
Noted 📔 👍